pkg/data/management/bindings_cleanup.go

```diff
@@ -8,10 +8,11 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
```

Comment:
Can we remove this blank line and sort imports?

Reply:
sure